pkgs/applications/display-managers/slim/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, fetchpatch, cmake, pkgconfig, xorg, libjpeg, libpng
+, fontconfig, freetype, pam, dbus, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "slim-ng";
+  version = "1.3.7";
+
+  src = fetchFromGitHub {
+    owner = "oxij";
+    repo = pname;
+    rev = "b6d77d41eeb796d3788ecab2d02e616c858661e1";
+    sha256 = "1rvrppc7n57d18022v7hmv3vglhsyb1nkql5qw6ka48wrrb2vcj7";
+  };
+
+  patches = [
+    # Allow the paths of the configuration file and theme directory to
+    # be set at runtime.
+    ./runtime-paths.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace /lib $out/lib
```

Comment:
Aargh, why does not the build script use [`GNUInstallDirs`](https://cmake.org/cmake/help/v3.16/module/GNUInstallDirs.html) CMake module?

Reply:
The patch is ugly too. The build system should work in a really native way if it's literally a "For NixOS" package. `GNUInstallDirs` would be the way to go for that.
(though, not very fond of CMake)